client/app/components/dynamic-parameters/DynamicButton.jsx

```diff
@@ -42,7 +43,7 @@ function DynamicButton({ options, selectedDynamicValue, onSelect, enabled, stati
 
   return (
     <div ref={containerRef}>
-      <a onClick={e => e.stopPropagation()}>
+      <Link role="presentation" onClick={e => e.stopPropagation()}>
```

Comment:
It feels to me that the `Link` component should only be used to what are real links (cc: @kravets-levko). Ideally it's used for event tracking and for any link manipulation globally.

@rafawendel sorry if I'm missing something, but why not keep the `<a>` here?

Reply:
Yes, exactly. For `<a>`s w/o `href` we probably should use `<Button type="link"`